src/main/java/uk/gov/laa/ccms/caab/controller/AssessmentController.java

```diff
@@ -46,18 +49,24 @@ public class AssessmentController {
   private final SecurityUtils contextSecurityUtil;
 
   @Value("${laa.ccms.oracle-web-determination-server.url}")
-  private final String owdUrl;
+  protected String owdUrl;
```

Comment:
Should these not be final, like they used to be, with lombok copying the annotations to the constructor?

Reply:
Without that it wasnt possible to test those variables existed when doing:

assessmentGet_validAssessment